src/Java.Interop.Dynamic/Java.Interop.Dynamic/JavaMethodInfo.cs

```diff
@@ -52,7 +52,7 @@ protected override void Dispose (bool disposing)
 
 		protected override string JniReturnType {
 			get {
-				if (ReturnType == null)
+				if (ReturnType == null || ReturnType.Name == "V")
```

Comment:
This was needed as `ReturnType` was not null, but "V" and thus the resulting member method was returning `LV;` instead of just `V` because the line below tried to do things.